spec/models/post_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'is invalid without a title' do
+    post = Post.new(title: nil, text: 'This is a test post.')
+    expect(post).to_not be_valid
+  end
+
+  it 'is invalid with a title exceeding 250 characters' do
+    post = Post.new(title: 'a' * 251, text: 'This is a test post.')
+    expect(post).to_not be_valid
+  end
```

Comment:
- [ ] You only added validations for post title presence and length, Please also add test cases for the validation of  `likes_counter` and `comments_counter`.

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=equal%20to%20zero.-,Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.,-Need%20a%20big), Please add unit tests for `five_recent_comments` and `update_posts_counter` methods because you forgot to add them.